launch/rgb8.launch

```diff
@@ -1,18 +1,18 @@
+<?xml version="1.0"?>
 <launch>
   <node name="check_ueye_api" pkg="ueye_cam" type="check_ueye_api" required="true" />
-  <arg name="nodelet_manager_name" value="nodelet_manager" />
-  <arg name="camera_name" value="camera" />
+  <arg name="nodelet_manager_name" default="nodelet_manager" />
```

Comment:
I don't have an issue with changing `nodelet_manager_name` into a default param too; please make it a consistent change for other sample roslaunch files too.

Reply:
done - mostly, I replaced all "value" by "default" for the declaration of the <arg>, so that it can be modified at execution time.